kytos/cli/commands/napps/api.py

```diff
@@ -200,12 +200,14 @@ def list(cls, args):
         napps = [napp + ('[ie]',) for napp in mgr.get_enabled()]
         napps += [napp + ('[i-]',) for napp in mgr.get_disabled()]
 
-        # Sort, add description and reorder coloumns
+        # Sort, add description and reorder columns
         napps.sort()
         napps_ordered = []
         for user, name, status in napps:
-            napps_ordered.append((status, '{}/{}'.format(user, name),
-                                  mgr.get_description(user, name)))
+            description = mgr.get_description(user, name)
+            version = mgr.get_version(user, name)
```

Comment:
This may lead to problems.

If no version was defined on `kytos.json`, or even if `kytos.json` does not have the `version` key, `mgr.get_version(...)` will return an empty string (`''`). Thus, version will not be set and will not be None.
Considering this, `napp_id` will be `user/name-`, which does not make sense.
@cemsbr, does my comment make sense?

Or get_version returns 'latest' if no version was found, or here we need to check the get_version output and format napp_id only if a version was defined.